<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Arya Bakhtiar</title>

  <meta name="author" content="Jon Barron">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="google-site-verification" content="K9KgflCc4PXSuzi-3eiiT6miTv4jD6uq5sGknNQjSKs"/>

  <link rel="stylesheet" type="text/css" href="stylesheet.css">
  <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>üåê</text></svg>">
</head>

<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <name>Arya Bakhtiar</name>
              </p>
              <p>
                I am student at UC Berkeley studying in the college of <a href="https://eecs.berkeley.edu">Electrical Engineering and Computer Sciences (EECS)</a> and <a href="https://mcb.berkeley.edu/">Molecular-Cellular Biology (MCB)</a>.
              </p>
              <p>
                At Berkeley, I am currently doing research in the <a href="https://bair.berkeley.edu/">Berkeley Artificial Intelligence Research (BAIR) Lab</a>, working with <a href="https://www.amirbar.net/">Amir Bar, Ph.D.</a> and <a href="https://people.eecs.berkeley.edu/~trevor/">Professor Trevor Darrell</a>. 
                I am also in the <a href="https://bayen.berkeley.edu/home">Mobile Sensing Lab</a>, working with <a href="https://humnetlab.berkeley.edu/index.php/people/alben-bagabaldo/">Alben Bagabaldo, Ph.D.</a> and <a href="https://bayen.berkeley.edu/alex-bayen"> Professor Alexandre Bayen</a>.
                In addition, I am a part-time research intern at the <a href="https://anlab.stanford.edu/">Newman Lab</a>, collaborating with <a href="https://profiles.stanford.edu/jeremy-dsilva">Jeremy D'Silva, Ph.D.</a> and <a href="https://med.stanford.edu/profiles/aaron-newman">Professor Aaron Newman</a>. 
                Previously, I was an undergraduate researcher at the Sandler Neuroscience Building at UCSF, where I was advised by <a href="https://henrylab.ucsf.edu/">Dr. Roland Henry</a>. 
              </p>
              <p style="text-align:center">
                <a href="mailto:bakhtiararya@berkeley.edu">Email</a> &nbsp/&nbsp
                <a href="https://github.com/bakhtiararya">Github</a> &nbsp/&nbsp
                <a href="https://www.linkedin.com/in/arya-bakhtiar-5563642a3/">LinkedIn</a> &nbsp/&nbsp
                <a href="https://twitter.com/aryabakhtiar777">Twitter</a>
              </p>
            </td>
            <td style="padding:2.5%;width:40%;max-width:40%">
              <a href="images/profile_pic.jpeg"><img style="width:100%;max-width:100%" alt="profile photo" src="images/profile_pic.jpeg" class="hoverZoomLink circle"></a>            
            </td>
          </tr>
        </tbody></table>
        
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Current Research Interests</heading>
              <p>
                I am fascinated by the intersection of biological data and advanced computational techniques, with a specific focus on generative modeling and self-supervised representation learning. 
                This passion drives my pursuit of innovative approaches to analyzing and interpreting complex biological datasets, aiming to uncover novel insights and applications in the realms of bioinformatics and personalized medicine.
              </p>
            </td>
          </tr>
        </tbody></table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Published Research Work</heading>
              <p>
                A couple of my past research work that are (or soon to be) publicly published and available to see:
              </p>
            </td>
          </tr>
        </tbody></table>

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td style="width: 35%; text-align: center">
              <div class="one" style="width: 85%; margin: auto">
              <img src="images/egopet.png" style="width: 100%; padding-top: 20px" />
              </div>
              <script type="text/javascript">
                function lvm_start() {
                  document.getElementById('lvm').style.opacity = "1";
                }
                function lvm_stop() {
                  document.getElementById('lvm').style.opacity = "0";
                }
              </script>
            </td>
            <td valign="middle" width="75%">
              <p>
                <a href="https://openreview.net/forum?id=OITfIp8mC2">
                <papertitle>EgoPet: A pet‚Äôs-eye view of the world for learning animal behavior</papertitle> </a>
                <br>
                 Amir Bar, <b>Arya Bakhtiar</b>, Antonio Loquercio, Jathushan Rajasegaran, Danny Tran, Yann LeCun, Amir Globerson, Trevor Darrel
                <br>
                <em>Submitted to CVPR 2024</em> 
                <br>
                Project Page (Coming Soon) | Code (Coming Soon)
                <br>
                <p> I co-founded EgoPet, an extensive dataset comprising over 84 hours of egocentric videos activities and behaviors of various animals. 
                  This project involved tasks such as object detection and frame assessment, crucial for improving locomotive performance in the Unitree A1 robotic dog and mirroring animal intelligence and interactions. 
                  Our findings and methodologies culminated in a publication submitted to the IEEE / CVF Computer Vision and Pattern Recognition Conference (CVPR).
                </p>
                </td>
          </tr>

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td style="width: 35%; text-align: center">
              <div class="one" style="width: 85%; margin: auto">
              <img src="images/trafficflow.png" style="width: 100%; padding-top: 20px" />
              </div>
              <script type="text/javascript">
                function lvm_start() {
                  document.getElementById('lvm').style.opacity = "1";
                }
                function lvm_stop() {
                  document.getElementById('lvm').style.opacity = "0";
                }
              </script>
            </td>
            <td valign="middle" width="75%">
              <p>
                <a href="https://bayen.berkeley.edu/research">
                <papertitle>Empirical evaluation of diverted traffic by signal coordination</papertitle> </a>
                <br>
                 Alben Bagabaldo, <b>Arya Bakhtiar</b>, Preston Fu, Alexander M. Bayen
                <br>
                <em>Publication in Progress</em>, Janurary 2024
                <br>
                Project Page (Coming Soon) | Code (Coming Soon)
                <br>
                <p> I investigated traffic signal timings in the Mission San Jose District of Fremont, California, using a validated and calibrated microscopic traffic simulation model considering 2019 demand data. 
                  Employing a dynamic traffic assignment microsimulation model with a logit-based route choice model, I examined the effects of both uncoordinated signal timings and the strategic reconfiguration of traffic signals on Mission Boulevard, one of Fremont's primary arterial roads.
                  Through my analysis, I discovered strategic reconfigurations of certain signal coordination strategies that could significantly redirect traffic flow, effectively alleviating congestion at targeted intersections and enhance the overall network performance. 
                </p>
                </td>
          </tr>

          <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td style="width: 35%; text-align: center">
              <div class="one" style="width: 85%; margin: auto">
              <img src="images/surf_mri.png" style="width: 100%; padding-top: 20px" />
              </div>
              <script type="text/javascript">
                function lvm_start() {
                  document.getElementById('lvm').style.opacity = "1";
                }
                function lvm_stop() {
                  document.getElementById('lvm').style.opacity = "0";
                }
              </script>
            </td>
            <td valign="middle" width="75%">
              <p>
                <a href="https://henrylab.ucsf.edu/">
                <papertitle>SLASER & JDE Analysis For MRI</papertitle> </a>
                <br>
                 <b>Arya Bakhtiar</b>, Roland Henry, Amit Akula
                <br>
                <em>Presented portion of research at the Berkeley Summer SURF 2019 Conference Event</em>
                <br>
                <a href="https://research.berkeley.edu/surf-fellows/arya-haghighi/">Project Page</a> |  <a href="https://github.com/bakhtiararya/ms_metabolica/tree/main">Code</a>
                <br>
                <p> I won a summer SURF fellowship by submitting a proposal to pursue an independent project. I developed software to track the concentration of specific metabolites, such as glutathione, using MRI images collected from patients diagnosed with Multiple Sclerosis. I presented part of my research findings to a summer research cohort. 
                </p>
                </td>
          </tr>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

        </tbody></table>

        <br><br><br>
            
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>On-going Research Projects</heading>
              <p>
              Some projects that I am continuing to work on:
              </p>
            </td>
          </tr>
        </tbody></table>

      <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td style="width: 35%; text-align: center">
              <div class="one" style="width: 85%; margin: auto">
              <img src="images/sample.png" style="width: 100%; padding-top: 20px" />
              </div>
              <script type="text/javascript">
                function lvm_start() {
                  document.getElementById('lvm').style.opacity = "1";
                }
                function lvm_stop() {
                  document.getElementById('lvm').style.opacity = "0";
                }
              </script>
            </td>
            <td valign="middle" width="75%">
              <p>
                <a href="https://anlab.stanford.edu/">
                <papertitle>Utilizing Graph Neural Networks with Spatial Transcriptomics Data</papertitle> </a>
               <br>
                 <b>Arya Bakhtiar</b>, Jeremy D'Silva
                <br>
                <em></em>Summer 2023 - Current<br>
                <br>
                <p>  I am investigating the spatial organization of distinct cell types in the tumor microenvironment and its impact on cell state and tumor progression. Utilizing Graph Neural Networks (GNNs), I am modeling complex cell interactions within tumors, 
                  capturing nuances in cellular behavior. My work primarily employs PyTorch Geometric, a tool for deep learning on graph-structured data, to analyze and interpret these cellular networks. 
                  Through this approach, I aim to unravel the intricate dynamics of tumor cells and their environment, contributing to advancements in cancer treatment strategies. 
                </p>
                </td>
          </tr>


        
       <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td style="width: 35%; text-align: center">
              <div class="one" style="width: 85%; margin: auto">
              <img src="images/gloscope.png" style="width: 100%; padding-top: 20px" />
              </div>
              <script type="text/javascript">
                function lvm_start() {
                  document.getElementById('lvm').style.opacity = "1";
                }
                function lvm_stop() {
                  document.getElementById('lvm').style.opacity = "0";
                }
              </script>
            </td>
            <td valign="middle" width="75%">
              <p>
                <a href="https://www.stat.berkeley.edu/~epurdom/research.html">
                <papertitle>Exploring scRNA-Seq Data at Population Scale with GloScope</papertitle> </a>
               <br>
                 <b>Arya Bakhtiar</b>, Yiyan Hao, Will Torous, Hao Wang, Elizabeth Purdom
                <br>
                <em></em>Fall 2023 - Current<br>
                <br>
                <p> I am conducting data extraction and enhancement on single-cell RNA sequencing (scRNA-Seq) information sourced from academic publications that studied diverse patient groups. 
                  Utilizing both R and Python, I am developing scripts to perform comprehensive analyses on how individual cell differences influence patient characteristics. 
                  In addition, I am working in close partnership with two doctoral candidates, taking an active role in periodic updates for the GlyScope initiative.
                </p>
                </td>
          </tr>



        
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

        </tbody></table>

        <br><br><br>
            
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Past Research Projects</heading>
              <p>
              </p>
            </td>
          </tr>
        </tbody></table>

      <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td style="width: 35%; text-align: center">
              <div class="one" style="width: 85%; margin: auto">
              <img src="images/osnmf.png" style="width: 100%; padding-top: 20px" />
              </div>
              <script type="text/javascript">
                function lvm_start() {
                  document.getElementById('lvm').style.opacity = "1";
                }
                function lvm_stop() {
                  document.getElementById('lvm').style.opacity = "0";
                }
              </script>
            </td>
            <td valign="middle" width="75%">
              <p>
                <a href="https://abbasilab.org/">
                <papertitle>Uncovering Biological Insights in Spatial Transcriptomics Data in Mouse Brain Aging</papertitle> </a>
               <br>
                 <b>Arya Bakhtiar</b>, Reza Abbasi-Asl
                <br>
                <em>Presented portion of research to lab seminar at Stanford</em>, Spring 2023<br>
                <a href="https://abbasilab.org/research/">Project Page</a> | <a href="https://github.com/bakhtiararya/mousebrain/tree/main">Code</a>
                <br>
                <p> At the Abbasi Lab, I leveraged deep autoencoders, non-negative matrix factorization (NMF), and semi-supervised deep CNNs to uncover significant and previously undiscovered spatial genomic patterns in complex brain images, which included a range of modalities 
                  such as fMRI and gene expression data. Additionally, I designed and validated interpretable machine learning frameworks and statistical tools, which revolutionized our understanding of cell types, their spatial arrangement, local gene networks, 
                  and revealed new patterns in complex brain images across human and mouse models, significantly advancing our knowledge of brain function, development, aging, and their implications for patient health and clinical decision-making. 
                </p>
                </td>
          </tr>

         <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td style="width: 35%; text-align: center">
              <div class="one" style="width: 85%; margin: auto">
              <img src="images/pathfx.png" style="width: 100%; padding-top: 20px" />
              </div>
              <script type="text/javascript">
                function lvm_start() {
                  document.getElementById('lvm').style.opacity = "1";
                }
                function lvm_stop() {
                  document.getElementById('lvm').style.opacity = "0";
                }
              </script>
            </td>
            <td valign="middle" width="75%">
              <p>
                <a href="https://research.seas.ucla.edu/computational-systems-pharmacology/">
                <papertitle> Utilized and Improved PathFX pipeline at the Lab for the Understanding of Network Effects (LUNE) </papertitle> </a>
                <br>
                <b>Arya Bakhtiar</b>, Jennifer L. Wilson
                <br>
                <a href="https://www.pathfxweb.net/index.php">Project Page</a> | <a href="https://github.com/bakhtiararya/pathfx_support">Code</a>
                <br>
                <p> I initiated a research project with Dr. Jenifer Wilson at UCLA using PathFX, a data-driven algorithm that predicts drug adverse effects using protein-protein interaction networks. 
                  After training against several tissue and cell type specific models, I developed modifications to the algorithmic pipeline that significantly enhanced PathFX‚Äôs capacity to predict drug-induced side effects. 
                  My analytical efforts led to the reevaluation of several drug pathways, a crucial step in unveiling new therapeutic strategies and drug repurposing. 
                </p>
                </td>
          </tr>

            
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>

        </tbody></table>

        <br><br><br>

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20"><tbody>
          <tr>
            <td>
              <heading>Teaching</heading>
               <p>
                 During my time at UC Berkeley, I served as a AI and course assistant tutor for several of the EECS and MCB courses. 
                 I spent 5 semesters assisting MCB C100A: Biophysical Chemistry - Physical Principles and the Molecules of Life for  <a href="https://chemistry.berkeley.edu/faculty/chem/kuriyan">Professor John Kuriyan</a> (now at Vanderbilt University) and <a href="https://vcresearch.berkeley.edu/faculty/david-savage">Professor David Savage</a>.  
              </p>
            </td>
          </tr>
        </tbody></table>
            
        <table width="100%" align="center" border="0" cellpadding="20"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/berkeley_eecs.jpg" alt="EECS" class="icon" width="200px" height="auto">
            </td>
            <td width="75%" valign="center">
              <p>
                <b>Data C100: Principles and Techniques of Data Science</b>
                <br>
                Course AI Assistant: <a href="https://ds100.org/">Course Website</a>
                <br>
              </p>
              <p>
                <b>CS 70: Discrete Mathematics and Probability Theory</b>
                <br>
                Course AI Assistant: <a href="https://www.eecs70.org/">Course Website</a>
              </p>
            </td>
          </tr>
        </tbody></table>

        <table width="100%" align="center" border="0" cellpadding="20"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/mcb.jpeg" alt="MCB" class="icon" width="200px" height="auto">
            </td>
            <td width="75%" valign="center">
              <p>
                <b>MCB C100A: Biophysical Chemistry - Physical Principles and the Molecules of Life</b>
                <br>
                Course TA Assistant: <a href="https://mcb.berkeley.edu/undergrad/courses/courses/ud-courses">Course Description</a>
                <br>
              </p>
            </td>
          </tr>
        </tbody></table>

        </tbody></table>
        
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:0px">
              <p style="text-align:right;font-size:small;">
                <br>
                Website template taken from <a href="https://jonbarron.info">Jon Barron</a>.
              </p>
            </td>
          </tr>
        </tbody></table>
      </td>
    </tr>
  </table>
</body>

</html>
